CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+## [0.7.1-rc8]
```

Comment:
Should we all be following that version pattern on other packages?

Reply:
I think it depends on how much there is in the package itself, if most of the logic is in the "other" package and on Flame I think this is a pretty good versioning system meanwhile we have the rc's, because then you know what it works with.